sentry_auth_github/client.py

```diff
@@ -51,8 +52,8 @@ def get_user_emails(self, access_token):
 
     def is_org_member(self, access_token, org_id):
         org_list = self.get_org_list(access_token)
-        org_id = str(org_id)
+        org_id = six.binary_type(org_id)
```

Comment:
I feel like `text_type` might be better here?

Reply:
Yes, this is really really bad in Python 3.

```
>>> bytes(10)
b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
```

Reply:
Oh, I remember seeing some of that junk during python 3 hackweek. Just following [sentry-flake8's advice](https://github.com/getsentry/sentry-flake8/blob/master/sentry_check.py#L442) though and https://six.readthedocs.io/#six.binary_type.

Reply:
Matt wanna throw up a sentry-flake8 pr?

Reply:
Well, the error is accurate. We were using it wrong before. `str == bytes` in python 2.

The correct equivalent way would be `unicode(org_id)`. This is a fundamental change in behavior. I noted this in the Notion doc.

Reply:
Wait, but `unicode` doesn't exist in Python 3. So you'd want `six.text_type` for py2 unicode and py3 str.

Reply:
Correct. I'm saying using `str(int)` in Python 2 is technically wrong. The lint conversion is correct since str == bytes. But in Python 3, this is not equivalent. In Python 2, the correct way to convert an integer to a string is with `unicode(int)`.

Reply:
Ohhh, I see.